nixos/tests/homebox.nix

```
@@ -0,0 +1,19 @@
+import ./make-test-python.nix (
+  {pkgs, ...}: {
+    name = "homebox";
+    meta = with pkgs.lib.maintainers; {
+      maintainers = [patrickdag];
+    };
+    nodes.machine = {
+      services.homebox.enable = true;
+    };
+    testScript = ''
+      machine.wait_for_unit("homebox.service")
+      machine.wait_for_open_port(7745)
+
+      out = machine.succeed("curl --fail -X GET 'http://localhost:7745/'")
```

You're not using the output.

```suggestion
      machine.succeed("curl --fail -X GET 'http://localhost:7745/'")
```

---

Also, I'd make the `7745` port more explicit as part of the test IMO (`let port = ...; in ...`, services.homebox.settings.HBOX_WEB_PORT = port`, etc...).

---

The actual content is rendered by the backend, so this return just a small standart nuxt page, which does not contain anything usefull to determin if it works as intended beyond being reachable.
I've explicitly set the port now.